client/src/components/EpisodeList/EpisodeList.css

```diff
@@ -1,8 +1,24 @@
 li {
   list-style-type: none;
-  padding-bottom: 1em;
+  padding: 0.5em;
+  font-family: 'Signika', sans-serif;
+    width: 95%;
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+    transition: 0.3s;
+    margin: 2%
+}
+
+li:hover {
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.5);
+    background-color: #FFC86D;
+
 }
 
 audio {
   float: right;
+  width: 45%;
+}
+
+ul {
+  padding: 6%;
 }
```

Comment:
Don't forget we have the tailwind utilities. 

https://tailwindcss.com/docs/spacing/

Reply:
Thanks, exploring that now.. because some traditional css doesn't work